src/native/minipal/cpufeatures.c

```diff
@@ -458,6 +462,11 @@ int minipal_getcpufeatures(void)
 
     // TODO: IsProcessorFeaturePresent doesn't support LRCPC2 yet.
 
+    if (IsProcessorFeaturePresent(PF_ARM_SVE_INSTRUCTIONS_AVAILABLE))
+    {
+        result |= ARM64IntrinsicConstants_Sve;
```

Comment:
Does this give us parity between Linux/Windows at this point or are there others that are still missing on Windows that Linux currently handles?

Reply:
`ARM64IntrinsicConstants_Rcpc2` and `ARM64IntrinsicConstants_Rdm` are still missing and I didn't see their equivalent yet on windows side.

Reply:
[LRCPC2 is supported since Windows 11 22H2.](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-isprocessorfeaturepresent#remarks)

Reply:
> [LRCPC2 is supported since Windows 11 22H2.](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-isprocessorfeaturepresent#remarks)

I don't see an entry of LRCPC2 in that. It just has LRCPC that is mapped to 45.

Reply:
Yes, it's not LRCPC2. Not a big deal though, it's just a minor optimization on top of RCPC1